tools/delete_candidate.php

```diff
@@ -51,12 +51,30 @@
 $candidate = new Candidate();
 $candidate->select($DCCID); //find the candidate with the given DCCID
 
+//Find Issues id with candidate frorign key
+$issueIDs = $DB->pselect("SELECT issueID 
+                FROM issues 
+                WHERE candID=:candID",
+                array('candID' => $DCCID)
+		);
+
+//delete issues_comments
+//delete issues_history
+foreach ($issueIDs as $issueID) {
```

Comment:
Well, we could use a join. But if it work that way, there is no need to maybe-break-it just for efficience.